cmd/lakectl/cmd/abuse.go

```diff
@@ -64,7 +64,7 @@ var abuseRandomReadsCmd = &cobra.Command{
 		}
 		Fmt("read a total of %d keys from key file\n", len(keys))
 
-		generator := stress.NewGenerator(parallelism, stress.WithSignalHandlersFor(os.Interrupt, syscall.SIGTERM))
+		generator := stress.NewGenerator("read", parallelism, stress.WithSignalHandlersFor(os.Interrupt, syscall.SIGTERM))
```

Comment:
const for actions?

Reply:
Here I think const is not required as it is like print out message with `Printf`